### PR TITLE
kubeadm: add a caution about certs in HA document

### PR DESCRIPTION
**Pull request by neolit123**
adding this note as requested in https://github.com/kubernetes/kubernetes/pull/73093#issuecomment-456915875

/assign @ereslibre @fabriziopandini 
/sig cluster-lifecycle
/kind cleanup
/priority important-longterm

### CODE REVIEW

File: content/en/docs/setup/independent/high-availability.md [hunk 1]
```diff
@@ -223,6 +223,12 @@ SSH is required if you want to control all nodes from a single machine.
     done
     ```
 
+{{< caution >}}
+Copy only the certificates in the above list. kubeadm will take care of generating the rest of the certificates
+with the required SANs for the joining control-plane instances. If by mistake you copy all the certificates,
```
Comment (ereslibre) [inline content/en/docs/setup/independent/high-availability.md]:
```suggestion
with the required SANs for the joining control-plane instances. If you copy all the certificates by mistake,
```

Not a native, wouldn't this rephrase work better?

Comment (neolit123) [reply content/en/docs/setup/independent/high-availability.md]: sgtm

**Comment by fabriziopandini** [comment]
/lgtm
/approve
Thanks @neolit123 !

**Pull request merged**